apps/server/src/modules/learnroom/service/board-copy.service.spec.ts

```diff
@@ -116,7 +126,7 @@ describe('board copy service', () => {
 				const user = userFactory.build();
 				const taskCopy = taskFactory.build({ name: originalTask.name });
 
-				taskCopyService.copyTaskMetadata.mockReturnValue({
+				taskCopyService.copyTask.mockResolvedValue({
```

Comment:
The refactoring should also required that the test structure must be refactored from beforeEach to setup. We can not avoid it when we touch the code. Otherwise we refactor it never.

Reply:
We refactored so much code in this ticket... and specifically at this file it did not work to refactor beforeEach to beforeAll... like in the other tests.

Reply:
Fixed in BC-2417